plugins/woocommerce-admin/client/subscriptions/index.tsx

```diff
@@ -73,36 +107,34 @@ const GetStartedButton: React.FC< GetStartedButtonProps > = ( {
 	const { installAndActivatePlugins } = useDispatch( PLUGINS_STORE_NAME );
 
 	return (
-		<div className="wcpay-empty-subscriptions__button_container">
-			<Button
-				disabled={ isGettingStarted }
-				isBusy={ isGettingStarted }
-				isPrimary
-				onClick={ () => {
-					setIsGettingStarted( true );
-					recordEvent(
-						'wccore_subscriptions_empty_state_get_started_click',
-						{}
-					);
-					installAndActivatePlugins( [ 'woocommerce-payments' ] )
-						.then( () => {
-							/*
-							 * TODO:
-							 * Navigate to either newSubscriptionProductUrl or onboardingUrl
-							 * depending on the which treatment the user is assigned to.
-							 */
-							// eslint-disable-next-line no-console
-							console.log( 'It was a success!' );
-						} )
-						.catch( () => {
-							setIsGettingStarted( false );
-							setHasError( true );
-						} );
-				} }
-			>
-				{ __( 'Get started', 'woocommerce' ) }
-			</Button>
-		</div>
```

Comment:
This change looks larger than it actually is. So the two buttons were in the same wrapper container, I moved this div out of here and put in the `MainContent` component instead, see change directly below this one. Let me know if there's a better way to do this.